model.ipynb

```diff
@@ -45,7 +55,77 @@
     "        features: ndarray\n",
     "            Flat vector encoding each amino acid (N x 1)\n",
     "        '''\n",
-    "        "
+    "    \n",
+    "    def get_fold(self, fold_idx = [0,1,2,3,4]):\n",
+    "        '''\n",
+    "        Extracts the desired folds and concatenate them into a single list\n",
+    "        \n",
+    "        Parameters\n",
+    "        ----------\n",
+    "        fold_idx\n",
+    "            List of number (from 0 to 4) to specify folds\n",
+    "        \n",
+    "        Returns\n",
+    "        ----------\n",
+    "        features: ndarray\n",
+    "            Concatenated list of the desired folds\n",
+    "        '''\n",
+    "        data_fold = np.empty\n",
+    "        \n",
+    "        for f in fold_idx:\n",
+    "            fold = self.data['C00{}'.format(f)]\n",
+    "            data_fold = np.concatenate((data_fold, fold), axis=0)\n",
```

Comment:
Nice! Consider adding a flag (default parameter) for randomizing the order of the data points